src/call_builder.ts

```diff
@@ -155,6 +174,7 @@ export class CallBuilder<
         if (es.addEventListener) {
           es.addEventListener("message", onMessage.bind(this));
           es.addEventListener("error", onError.bind(this));
+          es.addEventListener("close", onClose.bind(this));
```

Comment:
I think depending on the browser, some implementations of `EventSource` will route all "close" messages to a dedicated handler. Other implementations of `EventSource` will route "close" messages to the `onmessage` handler. I have tried to support both cases.

Reply:
Ahh when I was testing with Chrome and Brave it never reached onMessage, but adding a "close" event listener works. Nice!